content/docs/multi-user-overview.md

```diff
@@ -0,0 +1,144 @@
++++
+title = "Multi-user Isolation Support"
+description = "Isolation of user created resources for convenience and organization"
+weight = 50
++++
+
+Kubeflow v0.6 release enables support for multi-user isolation of user created
+resources in a Kubeflow deployment. The primary purpose of this functionality
+is to enable multiple users to operate on a shared Kubeflow deployment without
+stepping on each others' jobs and resources. It provides the users with the
+convenience of clutter free browsing of notebooks, training jobs, serving
+deployments and other resources. The isolation mechanisms also prevent
+accidental deletion/modification of resources of other users in the deployment.
+Note that the isolation support in Kubeflow doesn't provide any hard security
+guarantees against malicious attempts by users to infiltrate other user's
+profiles.
+
+## Usage overview
+
+An administrator needs to deploy Kubeflow and configure the authentication.
+service for the deployment.  A user can log into the system and will by default
+be accessing their primary profile. Users have view and modify access to their
+primary profile. The owner of a profile can share access to the profile with
+another user in the system. When sharing the access to a profile, the owner can
+choose to share it as a view access only or view and modify access with the
+user.
+
+A user can select their active profile from the top bar on
+the Kubeflow Central Dashboard. Note that the user can only view the profiles
```

Comment:
We haven't given the central UI a name, so it's best not to use capital letters - it looks like a brand name and so people may get confused looking for that name on the UI.

Instead, the docs just refer to "the Kubeflow central dashboard". The same applies on line 129 below.

Reply:
updated all occurrences of central dashboard to be lowercased.